ci/headers-bash.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# Copyright 2017 DigitalOcean
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+read -r -d '' LICENSE <<EOF
+#!/bin/bash
+
+# Copyright 2017 DigitalOcean
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+EOF
+
+FILES=$(find . -name "*.sh" -not -path "./vendor/*")
+EXIT=0
+
+for FILE in $FILES; do
+        HEADERS=$(head -n 15 $FILE)
+        if [ "$HEADERS" != "$LICENSE" ]; then
+                echo "license headers not found in $FILE"
+		EXIT=1
```

Comment:
Any reason you went with using an `EXIT` var? I think it's simpler if we just `exit 1` here and get rid of the EXIT var cause we would `exit 0` by default when the script ends anyways. What do you think?

Reply:
@andrewsykim If we would just use `exit 1`, once the file without headers is found, it would exit the script. That means we would get only one name of the file without headers, for each run, even if there are more (e.g. if there are 10 files without headers, you would have to run script 10 times).

Using `EXIT` var, allows script to run through all files, but yet to finish with error code.

Reply:
hmm, good point!